files/scripts/healdata/heal-cedar-data-ingest.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import json
+import requests
+import os
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("--directory", "-d", help="Cedar Directory ID for registering ")
+parser.add_argument("--access_token", "-a", help="User access token")
+
+args = parser.parse_args()
+
+if not args.directory:
+    print("Directory ID is required!")
+	exit(1)
+if not args.access_token:
+	print("User access token is required!")
+	exit(1)
+
+dir_id = args.directory
+access_token = args.access_token
+
+token_header = {"Authorization": 'access_token ' + access_token}
+
+# Get the metadata from cedar to register
+cedar = requests.get("http://revproxy-service/cedar/get_directory/" + dir_id, headers=token_header)
+
+# If we get metadata back now register with MDS
+if cedar.status_code == 200:
+	print("successfully got records from cedar directory")
+
+	metadata_return = cedar.json()
+
+	for record in metadata_return:
+		cedar_record = record["Metadata Location"]["Metadata Location - Details"]["nih_application_id"]
```

Comment:
is this correct? because the existance of this function https://github.com/uc-cdis/cedar-wrapper-service/blob/d553f7a4f5fe5752f2abb17bef8c64afdda57f52/src/cedar.py#L155 and this file https://github.com/uc-cdis/cedar-wrapper-service/blob/master/field_mapping.json#L4, the data returned from CEDAR wrapper should already have this field mapped to `appl_id`